docs/pages/sidenavigation.js

```diff
@@ -0,0 +1,638 @@
+        <MainSection.Subsection
+          title={generatedDocGen.SideNavigationItem?.displayName}
+          description={generatedDocGen.SideNavigationItem?.description}
+        >
+          <GeneratedPropTable
+            Component={SideNavigation?.Item}
+            name={generatedDocGen.SideNavigationItem?.displayName}
+            id={generatedDocGen.SideNavigationItem?.displayName}
+            generatedDocGen={generatedDocGen.SideNavigationItem}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Variants">
+        <MainSection.Subsection title="Sections">
+          <MainSection.Card
+            shaded
+            defaultCode={`
+<Box height={300} overflow="scroll">
+  <SideNavigation accessibilityLabel="Sections example">
+    <SideNavigation.Section label="Resources">
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/eslint_plugin"
+        item={{ label: 'Eslint plugin', value: 'eslint' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/faq"
+        item={{ label: 'FAQ', value: 'faq' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/how_to_hack_around_gestalt"
+        item={{ label: 'How to hack around Gestalt', value: 'hack' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/tooling"
+        item={{ label: 'Tooling', value: 'tooling' }}
+      />
+    </SideNavigation.Section>
+    <SideNavigation.Section label="Foundations">
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/accessibility"
+        item={{ label: 'Accessibility', value: 'accessibility' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/elevation"
+        item={{ label: 'Elevation', value: 'elevation' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/typography"
+        item={{ label: 'Typography', value: 'typography' }}
+      />
+      <SideNavigation.Item
+        href="https://gestalt.pinterest.systems/color_palette"
+        item={{ label: 'Color palette', value: 'color' }}
+      />
+    </SideNavigation.Section>
+  </SideNavigation>
+</Box>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Header">
+          <MainSection.Card
+            shaded
+            defaultCode={`
+function Example() {
+  const [organisedBy, setOrganisedBy] = React.useState('categorized');
+
+  return (
+    <Box height={300} overflow="scroll">
+      <SideNavigation
+        accessibilityLabel="Header example"
+        header={
+          <RadioGroup legend="Sort by?" id="example-header">
+            <Flex gap={2}>
+              <RadioGroup.RadioButton
+                checked={organisedBy === 'categorized'}
+                id="category"
+                label="Category"
+                name="sidebarSort"
```

Comment:
these are showing as 3 of 4 and 4 of 4, make sure the name is unique

Reply:
DONE